package.json

```diff
@@ -60,6 +60,7 @@
     "@taquito/utils": "19.0.2",
     "@temple-wallet/dapp": "5.0.2",
     "@temple-wallet/jest-webextension-mock": "^4.1.0",
+    "@temple-wallet/save-remote-file-webpack-plugin": "^1.1.2",
```

Comment:
I'm not sure that it will pass `npm audit` because the last publish was 4 years ago. Sorry, I don't want to do the same job as for `save-remote-file-webpack-plugin` again

Reply:
Unfortunately, I will have to make a fork of this package and, maybe, even modify its code because it is tailored to Webpack V4, see https://github.com/anzai9/fetch-write-webpack-plugin/blob/master/package.json#L69